src/database/sqlite/mod.rs

```diff
@@ -2101,6 +2050,14 @@ CREATE TABLE labels (
 
+        let bitcoin_txs: Vec<_> = (0..2)
```

Comment:
This `v0_to_v2_migration()` test should either be renamed or merged with the `v0_to_v5_migration()` test.

Reply:
True, it's clearly not migrating from v0 to v2.

Reply:
I've now merged it with the `v0_to_v5_migration()` test.